invenio_cli/helpers/scripts.py

```diff
@@ -233,6 +221,62 @@ def setup(dev=True,  force=False, docker_helper=None,
                 message="Assigning superuser access to admin role...",
                 verbose=verbose)
 
+
+def _setup_prod(force, docker_helper, project_shortname):
+    # Clean things up
+    if force:
+        click.secho("Flushing redis cache...", fg="green")
+        docker_helper.execute_cli_command(
+            project_shortname,
+            'invenio shell --no-term-title -c "import redis; ' +
+            "redis.StrictRedis.from_url(app.config['CACHE_REDIS_URL'])" +
+            '.flushall(); print(\'Cache cleared\')"')
+        click.secho("Deleting database...", fg="green")
+        docker_helper.execute_cli_command(
+            project_shortname, 'invenio db destroy --yes-i-know')
+        click.secho("Deleting indexes...", fg="green")
+        docker_helper.execute_cli_command(
+            project_shortname, 'invenio index destroy --force --yes-i-know')
+        click.secho("Purging queues...", fg="green")
+        docker_helper.execute_cli_command(
+            project_shortname, 'invenio index queue init purge')
+
+    click.secho("Creating database...", fg="green")
+    docker_helper.execute_cli_command(
+        project_shortname, 'invenio db init create',)
+    click.secho("Creating indexes...", fg="green")
+    docker_helper.execute_cli_command(
+        project_shortname, 'invenio index init --force')
```

Comment:
Should this `--force` be here?

Reply:
True, the is a use case for it (adding new schemas) but is not our case.